scripts/cybershake/install_cybershake.py

```diff
@@ -70,6 +70,12 @@ def load_args(logger):
         action="store_true",
         help="Keep stations if they snap to the same grid-point",
     )
+    parser.add_argument(
+        "--no_check_vm",
+        action="store_false",
+        dest="check_vm",
+        help="Set this flag if you are generating VMs from the automated workflow",
+    )
```

Comment:
I personally find no_XXX switch with store_false option confusing. Can we make it positive, like "skip_vm_check" and store_true? You could be also a bit more explicit by setting the default value.

Reply:
The aim is to maintain existing functionality, hence why adding an argument to skip the vm checking was added.

The reason for the store_false is to avoid checks later that state `if not skip_vm_check` as that at a glance is a lot harder to read than `if check_vm`.

I'll add a default value for clarity.

Reply:
Sure - I was thinking we could still set check_vm = not skip_vm_check, and keep the rest of the code unchanged, but I can accept it as it is - nice to have the default value.